docs/appendices/release-notes/4.3.2.rst

```diff
@@ -0,0 +1,68 @@
+.. _version_4.3.2:
+
+=============
+Version 4.3.2
+=============
+
+Released on 2020-11-25.
+
+.. NOTE::
+
+    If upgrading a cluster, you must be running CrateDB 4.0.2 or higher before
```

Comment:
```suggestion
    When upgrading a cluster, you must be running CrateDB 4.0.2 or higher before
```

Reply:
We've this line in pretty much every release and I think that was also already reviewed by @mechanomi 

Not sure about the `if` vs `when` cases. I'll keep it as is for now.